lib/app.jsx

```diff
@@ -327,6 +327,13 @@ export const App = connect(
         content,
       });
 
+    onSyncNote = noteId => {
```

Comment:
I know we have precedent with all of our other actions, but `onSyncNote` seems misleading here. in these cases we're not responding to things but rather manually triggering them. did you consider naming the prop `syncNote`?